Fix the Exec line in the awsvpnclient.desktop file to use the correct path and escape spaces
--- awsvpnclient.desktop.orig	2025-11-10 09:52:45.000000000 +0100
+++ awsvpnclient.desktop	2025-11-10 09:52:45.000000000 +0100
@@ -3,9 +3,9 @@
 Encoding=UTF-8
 Name=AWS VPN Client
 Comment=AWS VPN Client
-Exec="/opt/awsvpnclient/AWS\sVPN\sClient" %u
+Exec="/opt/awsvpnclient/AWS VPN Client" %u
 Path=/opt/awsvpnclient
-Icon=acvc-64.png
+Icon=acvc-64
 Terminal=false
